posthog/migrations/0540_errortrackingteam_errortrackingteammembership_and_more.py

```diff
@@ -0,0 +1,111 @@
+    dependencies = [
+        ("posthog", "0539_user_role_at_organization"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ErrorTrackingTeam",
+            fields=[
+                (
+                    "id",
+                    models.UUIDField(
+                        default=posthog.models.utils.UUIDT, editable=False, primary_key=True, serialize=False
+                    ),
+                ),
+                ("name", models.TextField()),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+            ],
+            options={
+                "abstract": False,
+            },
+        ),
+        migrations.CreateModel(
+            name="ErrorTrackingTeamMembership",
+            fields=[
+                (
+                    "id",
+                    models.UUIDField(
+                        default=posthog.models.utils.UUIDT, editable=False, primary_key=True, serialize=False
+                    ),
+                ),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+            ],
+        ),
+        migrations.RemoveConstraint(
+            model_name="errortrackingissueassignment",
+            name="unique_on_user_and_issue",
+        ),
+        migrations.AlterField(
+            model_name="errortrackingissueassignment",
+            name="issue",
+            field=models.OneToOneField(
+                on_delete=django.db.models.deletion.CASCADE, related_name="assignment", to="posthog.errortrackingissue"
+            ),
+        ),
+        migrations.AlterField(
+            model_name="errortrackingissueassignment",
+            name="user",
+            field=models.ForeignKey(
+                null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL
+            ),
+        ),
+        migrations.AddField(
+            model_name="errortrackingissueassignment",
+            name="error_tracking_team",
+            field=models.ForeignKey(
+                null=True, on_delete=django.db.models.deletion.CASCADE, to="posthog.errortrackingteam"
+            ),
+        ),
+        AddConstraintNotValid(
+            model_name="errortrackingissueassignment",
+            constraint=models.CheckConstraint(
+                check=models.Q(("user__isnull", False), ("error_tracking_team__isnull", False), _connector="OR"),
+                name="at_least_one_non_null",
+            ),
+        ),
+        AddConstraintNotValid(
+            model_name="errortrackingissueassignment",
+            constraint=models.CheckConstraint(
+                check=models.Q(("user__isnull", False), ("error_tracking_team__isnull", False), _negated=True),
+                name="only_one_non_null",
+            ),
+        ),
+        migrations.AddConstraint(
+            model_name="errortrackingissueassignment",
+            constraint=models.UniqueConstraint(fields=("issue",), name="unique_per_issue"),
+        ),
```

Comment:
Some of these constraints are causing the checks to fail due to table locking. Given this table is tiny right now it feels easier to override the check than go about writing the custom up / down SQL by hand cc @neilkakkar

Reply:
yeah i think will be fine 👍